src/vstart.sh

```diff
@@ -128,6 +128,7 @@ if [ `uname` = FreeBSD ]; then
 else
     objectstore="bluestore"
 fi
+ceph_osd=ceph-osd
```

Comment:
i think it comes from 82404ca21dc59c1c369a8333b707f090dbadc6bf . but since crimson-osd is not ready for `mkfs` or creating PGs. i guess we can leave this for future?